src/tests/testTokenizer.cc

```diff
@@ -264,6 +543,36 @@ TestTokenizer::testTokenizerInt64()
         CPPUNIT_ASSERT_EQUAL(SBuf("  1234"), t.buf());
     }
 
+    // zero corner case: repeated zeros
+    {
+        int64_t rv;
+        Parser::Tokenizer t(SBuf("00"));
+        const int64_t benchmark = 00;
+        CPPUNIT_ASSERT(t.int64(rv));
+        CPPUNIT_ASSERT_EQUAL(benchmark,rv);
+        CPPUNIT_ASSERT_EQUAL(SBuf(""), t.buf());
+    }
+
+    // zero corner case: "positive" zero
+    {
+        int64_t rv;
+        Parser::Tokenizer t(SBuf("+0"));
+        const int64_t benchmark = +0;
+        CPPUNIT_ASSERT(t.int64(rv));
+        CPPUNIT_ASSERT_EQUAL(benchmark,rv);
+        CPPUNIT_ASSERT_EQUAL(SBuf(""), t.buf());
+    }
+
+    // zero corner case: "negative" zero
+    {
+        int64_t rv;
+        Parser::Tokenizer t(SBuf("-0"));
+        const int64_t benchmark = -0;
+        CPPUNIT_ASSERT(t.int64(rv));
+        CPPUNIT_ASSERT_EQUAL(benchmark,rv);
+        CPPUNIT_ASSERT_EQUAL(SBuf(""), t.buf());
+    }
+
```

Comment:
Don't we need also to test overflow with a zero-leading huge value?

Reply:
Yes, better overflow tests would be useful, but they are outside the current PR scope IMO. Writing a good black-box leading-zero overflow test is not trivial. Adding a white-box leading-zero overflow test is not needed -- int64() loop is currently not counting the number of digits in any way and, hence, does not care if there are leading zeros AFAICT. Let's not add those tests (in this PR).

P.S. I added (useful) `+0` and `-0` tests because adding them was _trivial_. They are arguably in this PR scope.